database/migrations/1503250034279_user.ts

```diff
@@ -8,7 +8,7 @@ export default class extends BaseSchema {
       table.increments();
       table.string('username', 80).notNullable();
       table.string('email', 254).notNullable().unique();
-      table.string('password', 60).notNullable();
+      table.string('password', 254).notNullable();
```

Comment:
Hi!

Thank you for raising this PR. I haven't tested this yet (been having some personal issues that prevent me from reviewing and testing stuff)....

Nevertheless, we shouldn't modify migrations file, we should rather create a new one and modify that field to allow up to 254. Would you be able to create a new migration following adonisjs guides and change this PR accordingly?

Thank you so much in advance!